src/lib/jobRunner.ts

```diff
@@ -11,13 +18,16 @@
  * @returns {Function} returns a function that accepts the parameters
  * to be passed to the "userFunc" function
  */
-const jobRunner = (userFunc: Function) => (e: MessageEvent) => {
+const jobRunner = (options: JOB_RUNNER_OPTIONS) => (e: MessageEvent) => {
   const [userFuncArgs] = e.data as [any[]]
-
-  return Promise.resolve(userFunc(...userFuncArgs))
+  return Promise.resolve(options.fn(...userFuncArgs))
     .then(result => {
+      const isTransferable = (val: any) => (
+        val instanceof ArrayBuffer || val instanceof MessagePort || val instanceof ImageBitmap
+      )
```

Comment:
I guess it is better to move this function to the `utils` to reuse in the `useWorker.ts` file, does it make sense?

Reply:
Since `jobRunner` will be converted into a `string` ( [createWorkerBlob:24](https://github.com/alewin/useWorker/pull/48/files#diff-6a152510d135a64b25685c5b8e68efb9R24)), I think we couldn't reuse `isTransferable` as a util :/

Reply:
Ah right, completely forgot about it, thanks :)